### ISSUE TITLE
ia11: multiple selectajax marks only the first stored value as selected — saving the form then wipes the rest of the relation

### ISSUE THREAD

**Issue by Pal9yni4bi** [LaravelRUS/SleepingOwlAdmin#1440]
## Branch

`ia11` (installed as `laravelrus/sleepingowl:dev-ia11`). Verified on `b3752374`.

## Summary

A multiple `selectajax` (`AdminFormElement::multiselectajax()`) renders only its **first** stored value as `selected`. Every other saved value is present in the `<select>` as an option, but unselected.

This is not only a display defect. The browser submits what is selected, and `ElementSaveRelation::afterSave()` syncs the relation with exactly that:

```php
$relation->sync($values);
```

So opening a record and pressing Save — without touching the field — **deletes every related row but one**. On our installation a user account holding 10 roles rendered 1 selected role; saving the form would have dropped the other 9.

## Root cause

`resources/views/default/form/element/selectajax.blade.php:13`:

```php
{!! html()->select($name, $options, $value)->attributes($attributes) !!}
```

`Html::select()` applies the value immediately (`Spatie\Html\Html::select()` → `Select::value()` → `Select::applyValueToOptions()`), and that method truncates the value set whenever the element is not marked multiple:

```php
protected function applyValueToOptions()
{
    $value = ...;

    if (! $this->hasAttribute('multiple')) {
        $value = $value->take(1);
    }

    $this->children = $this->applyValueToElements($value, $this->children);
}
```

At that point the element has no `multiple` attribute — it arrives one call later, via `->attributes($attributes)`. And `BaseElement::attributes()` only sets attributes; it never recomputes the selection. So `take(1)` wins permanently.

`MultiSelectAjax::toArray()` does put `multiple` into the attribute array, so the rendered tag is a correct `<select multiple>` — which is why the markup looks right while only one option carries `selected`.

This is a regression against the pre-`spatie/laravel-html` code. The same line used to read (still visible as a comment right above it):

```php
{!! Form::select($name, $options, $value, $attributes) !!}
```

`laravelcollective/html` received attributes and value in one call, so `multiple` was known before the selection was computed.

## Reproduction

```php
AdminFormElement::multiselectajax('roles')
    ->setLabel('Roles')
    ->setModelForOptions(\Spatie\Permission\Models\Role::class, 'name'),
```

1. Give a record several related rows (e.g. a user with 3 roles).
2. Open the edit form.
3. Expected: all 3 shown as selected in the select2 control.
4. Actual: 1 shown. Rendered markup — all options present, one `selected`:

```html
<select class="form-control js-data-ajax" name="roles[]" multiple>
  <option value="3" selected="selected">Role A</option>
  <option value="4">Role B</option>
  <option value="5">Role C</option>
</select>
```

5. Press **Save** without touching anything → the record is left with a single role.

`selectajax` (single) is not affected: it has no `multiple` attribute, so `take(1)` is the correct behaviour there.

## Proposed fix

Put `multiple` on the element before the value is applied. `Select::multiple()` re-runs `applyValueToOptions()` itself, so the stored value — including a value restored from old input — is re-applied against the full option list:

```php
@php($select = html()->select($name, $options, $value)->attributes($attributes))
{!! $select->hasAttribute('multiple') ? $select->multiple() : $select !!}
```

The single-select path is untouched: without a `multiple` attribute the ternary returns the element unchanged.

`Select::multiple()` also normalises the name to `foo[]`, which `MultiSelect::getName()` has already done — appending is guarded by a `Str::endsWith($name, '[]')` check, so nothing changes there.

A PR against `ia11` follows. Happy to port it to `development` too, the line is identical there.
